app/web_conn.go

```diff
@@ -380,6 +383,24 @@ func (wc *WebConn) readPump() {
 	}
 }
 
+func (wc *WebConn) Subscribe(id model.WebsocketSubjectID) {
+	wc.mu.RLock()
+	defer wc.mu.RUnlock()
```

Comment:
Btw, this is modifying the map, so we needed Lock/Unlock here :) Only the read case should use R(Lock/Unlock). Anyways, no longer needed now.